### PR TITLE
Ffmpeg check change

### PR DESCRIPTION
**Pull request by seb-b**

### CODE REVIEW

File: wagtailvideos/utils.py [hunk 1]
```diff
@@ -4,3 +4,9 @@
     from shutil import which
 except ImportError:
     from distutils.spawn import find_executable as which
+
+
+def ffmpeg_installed(path=None):
+    if which('ffmpeg', path=path) is None:
```
Comment (timheap) [inline wagtailvideos/utils.py]:
```python
return which('ffmpeg', path=path) is not None
```

File: wagtailvideos/apps.py [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
 
 
 def ffmpeg_check(app_configs, path=None, **kwargs):
-    errors = []
-    if which('ffmpeg', path=path) is None:
-        errors.append(
-            Error(
-                'ffmpeg could not be found on your system, try installing it.',
+    messages = []
+    if not ffmpeg_installed(path=path):
```
Comment (timheap) [inline wagtailvideos/apps.py]: `ffmpeg_check()` will never be called with a `path` argument by Django (at least according to the Django docs which make no mention of any arguments), so the default `None` value flows through the whole call chain to `which`. Can this parameter just be removed?

**Comment by timheap** [comment]
Can you add some tests that make a video without a thumbnail and check the admin still works, and the template tag behaves correctly?

`unittest.mock` may be helpful here for making `ffmpeg_installed()` return False. A backport to Python 2.7 is [available on PyPI](https://pypi.python.org/pypi/mock)

File: wagtailvideos/apps.py (hunk 2 above)
Comment (seb-b) [inline wagtailvideos/apps.py]: yeah not sure that what that was in there for I'll remove it

**Pull request merged**